Server/ConcordiaCurriculumManager/Controllers/GroupController.cs

```diff
@@ -127,4 +127,35 @@ public async Task<IActionResult> RemoveGroupMaster([FromRoute, Required] Guid gr
 
         return Ok();
     }
+
+    [HttpPut("{groupId}")]
+    [Authorize(Policies.IsGroupMasterOrAdmin)]
```

Comment:
I think for both methods it makes more sense to have this only available to admins, not group masters too.